webapp/i18n/en.json

```diff
@@ -27,15 +27,15 @@
   "BoardPage.newVersion": "A new version of Boards is available, click here to reload.",
   "BoardPage.syncFailed": "Board may be deleted or access revoked.",
   "BoardTemplateSelector.add-template": "Create new template",
-  "BoardTemplateSelector.create-empty-board": "Create empty board",
+  "BoardTemplateSelector.create-empty-board": "Create an empty board",
```

Comment:
This change is causing a cypress failure...

Reply:
That's really weird. I wonder why. I can revert the change if it'll help. 

The weblate changes are for non-en.json (@cwarnermm, just checking if that's correct?)

Reply:
All string changes should be made _only_ in the en.json file. All other language resource files for supported languages are populated through Weblate via translation contributions.

Reply:
That makes sense that a Cypress test could fail as a result of a string changing. The Cypress tests look for whatever it thinks the string is and if it doesn't find it, it throws an error.

When Docs modifies an existing product string, QA recommends we include a QA team member to review and ensure that associated tests are updated accordingly. Docs isn't required to maintain the Cypress tests, and QA very much appreciates a heads-up that a product change may impact their testing systems.

Reply:
Brilliant, thanks Carrie! Learned something new. :) I've added QA.